lightkurve/lightcurve.py

```diff
@@ -592,9 +600,121 @@ def to_csv(self, path_or_buf=None, **kwargs):
         """
         return self.to_pandas().to_csv(path_or_buf=path_or_buf, **kwargs)
 
+    def to_fits(self, path=None, overwrite=False, **kwargs):
+        """Writes the KeplerLightCurve to a fits file.
+
+        Parameters
+        ----------
+        path : string, default None
+            File path, if None returns an astropy.io.fits object.
+        overwrite : bool
+            Whether or not to overwrite the file
+        **kwargs : dict
+            Keywords to pass to to_fits. Keywords that are str, int, float or bool
+            are converted to headers. Keywords that are np.arrays or lists are
+            converted to columns.
+
+        Returns
+        -------
+        hdu : astropy.io.fits
+            Returns an astropy.io.fits object if path is None
+        """
+        typedir = {int: 'J', str: 'A', float: 'D', bool: 'L',
+                   np.int32: 'J', np.int32: 'K', np.float32: 'E', np.float64: 'D'}
+
+        def _header_template(extension):
+            """Returns a template `fits.Header` object for a given extension."""
+            template_fn = os.path.join(PACKAGEDIR, "data",
+                                       "lc-ext{}-header.txt".format(extension))
+            return fits.Header.fromtextfile(template_fn)
+
+        def _make_primary_hdu(keywords={}):
+            """Returns the primary extension (#0)."""
+            hdu = fits.PrimaryHDU()
+            # Copy the default keywords from a template file from the MAST archive
+            tmpl = _header_template(0)
+            for kw in tmpl:
+                hdu.header[kw] = (tmpl[kw], tmpl.comments[kw])
+            # Override the defaults where necessary
+
+            default = default = {'ORIGIN': "Unofficial data product",
+                                 'DATE': datetime.datetime.now().strftime("%Y-%m-%d"),
+                                 'CREATOR': "lightkurve"}
+
+            for kw in default:
+                hdu.header['{}'.format(kw).upper()] = default[kw]
+                if default[kw] is None:
+                    log.warn('Value for {} is None'.format(kw))
+            if ('quarter' in dir(self)) and (self.quarter is not None):
+                hdu.header['QUARTER'] = self.quarter
+            elif ('campaign' in dir(self)) and self.campaign is not None:
+                hdu.header['CAMPAIGN'] = self.campaign
+            else:
+                log.warn('Cannot find Campaign or Quarter number')
```

Comment:
I get: `DeprecationWarning: The 'warn' method is deprecated, use 'warning' instead`